commit e8077044df239bcf0d9e9980b0e1afb9f1f5c446
Author: fanquake <fanquake@gmail.com>
Date:   Tue Aug 18 20:50:19 2020 +0800

    Don't use wingenminiupnpcstrings when generating miniupnpcstrings.h

    The wingenminiupnpcstrings tool is used on Windows to generate version
    information. This information is irrelevant for us, and trying to use
    wingenminiupnpcstrings would cause builds to fail, so just don't use it.

    We should be able to drop this once we are using 2.1 or later. See
    upstream commit: 9663c55c61408fdcc39a82987d2243f816b22932.

diff --git a/Makefile.mingw b/Makefile.mingw
index 574720e..fcc17bb 100644
--- a/Makefile.mingw
+++ b/Makefile.mingw
@@ -74,7 +74,7 @@ wingenminiupnpcstrings:	wingenminiupnpcstrings.o

 wingenminiupnpcstrings.o:	wingenminiupnpcstrings.c

-miniupnpcstrings.h: miniupnpcstrings.h.in wingenminiupnpcstrings
+miniupnpcstrings.h: miniupnpcstrings.h.in
 	wingenminiupnpcstrings $< $@

 minixml.o:	minixml.c minixml.h
